elements/pfe-styles/demo/pfe-colors.css

```diff
@@ -466,7 +466,7 @@
     background: #e0d7ee;
   }
 
- .layouts .pfe-l-grid > * {
+ .layouts .pfe-l-grid:not(ol.pfe-list--primary) > * {
```

Comment:
I wonder if we can target all ol or ul tags in general 🤔

Reply:
Yeah, good thinking.  I would say we can probably target `ol,ul`.  That would make this code general and more maintainable too.